TACC/Core-Styles
# refactor: c-card--image-bottom

## Overview

Do not let image-top card's `<h3>` fill available space.

## Related

- fixes edge case bug found while working on #296 

## Changes

- **changed** image-top and image-bottom
    - to use same, simple `grid-template-rows` value
    - to not try to let client change image-height
- **documented** card pattern as "Ready" (not "Work in Progress")

## Testing

1. Open http://localhost:4000/components/preview/c-card--images.
2. Remove `o-grid--col-min-width`.
    <sup>This makes the card be an item in a grid with equal height rows.</sup>
3. Verify extra space on "Card - Image bottom" is beneath image.
    <sup>(**not** between title and description)</sup>

## UI

| Before | After |
| - | - |
| <img width="960" alt="before" src="https://github.com/TACC/Core-Styles/assets/62723358/ac28d06d-aea0-4ada-bfd6-7317a8d4e251"> | <img width="960" alt="after" src="https://github.com/TACC/Core-Styles/assets/62723358/8126b8d0-51db-46cc-9987-af699c3aa3ad"> |

## Notes

- The previous image-bottom `grid-tempalte-rows` value **incorrectly** assumed image was last row.
- Both image-top and image-bottom `grid-tempalte-rows` values **incorrectly** assumed we could reliably control image height.
